unlock-protocol.com/blog/unlocking-scout.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Unlocking Scout
+subTitle: Our friends at Scout have integrated Unlock on the Unlock dashboard. Here's how it works
```

Comment:
I wonder if there's another way to word this to avoid repeating "Unlock," it sounds a bit stilted to me